packages/react-composites/src/composites/ChatComposite/file-sharing/FileUploadButton.tsx

```diff
@@ -80,7 +61,7 @@ export const FileUploadButton = (props: FileUploadButtonProps): JSX.Element => {
         accept={accept}
```

Comment:
the stack this is in doesn't have a tabindex, is it keyboard navigatable? (a11y work could also be a seperate work item for this all)

Reply:
Not thinking about a11y work at this point. Will be tackling the entire file sharing UI from an a11y perspective later. :)